docs/api/item.md

```diff
@@ -28,6 +32,125 @@ import Basic from '@site/static/usage/v7/item/basic/index.md';
+/>
+
+<BestPracticeFigure 
+  text={<>Developers should also consider how important the metadata is. Drawing attention to the metadata may be helpful for the user or it may distract them from the more important information depending on the use case.</>}
+  doText={<>Prioritize the most important content.</>}
+  cautionText={<>Prioritized metadata may distract from other important content.</>}
+  doImage={<img alt="A list that contains several items, each representing a different to-do list. A count of how many tasks in each to-do list is placed at the end of each item." src={useBaseUrl('img/item/metadata-relevant-do.jpg')} />}
+  cautionImage={<img alt="A list that contains several items, each representing a different to-do list. A count of how many tasks in each to-do list is placed at the end of each item. However, the count is highlighted in blue which draws the user's attention away from the name of the to-do list." src={useBaseUrl('img/item/metadata-important-caution.jpg')} />}
+/>
+
+In the example below, we are creating two lists with different kinds of metadata. The first list uses [Note](./note) to show how many tasks are in each to-do list.
+
+The second list mimics the iOS Mail app to show an inbox. This list makes use of custom metadata including an "unread message" indicator in the "start" slot as well as a timestamp and custom detail icon in the "end" slot. The "unread message" indicator is highlighted in blue to draw the user's attention to the unread messages, while the timestamp is more subtle.
+
+import Metadata from '@site/static/usage/v7/item/content-types/metadata/index.md';
+
+<Metadata />
+
+### Actions
+
+Actions are interactive elements that do something when you activate them. An item can have multiple actions displayed on a line. However, developers should ensure that each action's tap target is large enough to be usable.
+
+<BestPracticeFigure 
+  text={<>Developers should avoid creating <a href="https://dequeuniversity.com/rules/axe/4.4/nested-interactive">nested interactives</a> which can break the user experience with screen readers. For example, developers should avoid adding a button inside the main content of the Item if the <code>button</code> property is set to <code>true</code>. Additionally, developers should use the <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a> component if they need to show more than two actions.</>}
+  doText={<>Avoid nested interactives and put multiple items in <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a>.</>}
+  dontText={<>Don't put too many actions on a single item.</>}
+  doImage={<img alt="A list that contains several items, each representing a different to-do list. A count of how many tasks in each to-do list is placed at the end of each item." src={useBaseUrl('img/item/actions-do.jpg')} />}
```

Comment:
The way the "do" image shows the item-sliding looks strange with the items on the next row like that; is there a way to screenshot it in the middle of the gesture instead? Or does it not look clear enough what's actually going on? 🤔

Reply:
My concern is people may think the item is supposed to look like that normally (mid-gesture with some buttons cut off). I updated the text to make it clear these items are revealed by swiping in https://github.com/ionic-team/ionic-docs/pull/3123/commits/b1a9c33b78326f19fa3f7d73be117cbf5b417e31. Does this help at all?

Reply:
Yeah, I think that'll have to do. Thank you!